core/api/src/graphql/public/root/query/ln-invoice-payment-status.ts

```diff
@@ -1,10 +1,10 @@
 import { Lightning } from "@/app"
+import { WalletInvoiceStatus } from "@/domain/wallet-invoices"
 
 import { GT } from "@/graphql/index"
-import { mapError } from "@/graphql/error-map"
+import { mapAndParseErrorForGqlResponse } from "@/graphql/error-map"
 import LnInvoicePaymentStatusPayload from "@/graphql/public/types/payload/ln-invoice-payment-status"
 import LnInvoicePaymentStatusInput from "@/graphql/public/types/object/ln-invoice-payment-status-input"
```

Comment:
Same comment here for whether we should be more specific internally
```suggestion
import LnInvoicePaymentStatusByRequestInput from "@/graphql/public/types/object/ln-invoice-payment-status-by-request-input"
```

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/3899/commits/0f67fbcf9456930df5d2d5fc09ebdb2dedd3f9f7 although name remains the same to avoid breaking changes `name: "LnInvoicePaymentStatusInput"`